lib/jsonApiTranslator.js

```diff
@@ -115,6 +115,13 @@ function transformQuery(req) {
         }
         return value
     }
+    function transformRelationships(key) {
+        const relationships = schema.relationships || {};
+        if(relationships[key]){
+            return `relationships.${key}.data.id`;
+        }
+        return key;
```

Comment:
seems like the ret types might mismatch here/be surprizing here, but this may be a misread.

Reply:
ok, your slack comment clears this up.